src/helpers/utilities.ts

```diff
@@ -151,6 +152,12 @@ export const convertStringToHex = (stringToConvert: string): string =>
 export const add = (numberOne: BigNumberish, numberTwo: BigNumberish): string =>
   new BigNumber(numberOne).plus(numberTwo).toFixed();
 
+export const addDisplay = (numberOne: string, numberTwo: string): string => {
```

Comment:
Not sure if we have a way to add sets of display values right now, so I added a util to do that (just using `add` then formatting a value to specific number of decimals can produce rounding issues).

Reply:
https://github.com/rainbow-me/rainbow/pull/2947/files#r806087085